samples/switchless/host/host.c

```diff
@@ -82,8 +82,8 @@ int main(int argc, const char* argv[])
         flags |= OE_ENCLAVE_FLAG_SIMULATE;
     }
 
-    // Enable switchless and configure host worker number
-    oe_enclave_setting_context_switchless_t switchless_setting = {1, 0};
+    // Enable switchless and configure 1 host and 1 enclave worker
```

Comment:
Expand comment... how were the numbers 1 and 1 chosen?

Reply:
I have provided the rationale/assumptions based on which the sample was written.
Hopefully it is more understandable for the user now.